distribution/lib/Standard/Base/0.2.32-SNAPSHOT/src/Data/Vector.enso

```diff
@@ -755,6 +755,9 @@ type Vector
              [Pair 1 "a", Pair 2 "b", Pair 1 "c"] . distinct (on = _.first) == [Pair 1 "a", Pair 2 "b"]
     distinct : (Any -> Any) -> Vector Any
     distinct (on = x->x) =
+        ## TODO [JD] This is based on the Map class until a HashMap is available.
+           Should be changed once https://www.pivotaltracker.com/story/show/181027272 is completed.
+
```

Comment:
This is described somewhere in our style guide - anyway, such comments should always explain to the reader:
- Why this change was done. 
- Why the other approach is better.
- Why the other approach is not available now.
- Link to in-depth explanation.

Basically, the idea is that reading the comment should provide the reader with enough information of what happened and the link should be in-depth read on-demand. However, the reader should be a blessing to skip clicking the link and still understand what happened in the code.